.github/workflows/ci.yml

```diff
@@ -186,6 +186,7 @@ jobs:
       - name: Run the script
         run: |
           pushd zero_bin/tools
+          mkdir proofs
```

Comment:
are these needed? Shouldn't we create the output dir by default if nonexistent?

Reply:
We don't I think (maybe for circuits we do). Will add check and create by default if nonexistent.